token/program-2022/src/extension/confidential_mint_burn/processor.rs

```diff
@@ -210,20 +210,36 @@ fn process_confidential_mint(
         }
     }
 
+    let proof_context_auditor_ciphertext_lo = proof_context
+        .mint_amount_ciphertext_lo
+        .try_extract_ciphertext(2)
+        .map_err(|e| -> TokenError { e.into() })?;
```

Comment:
nit-but-it's-up-to-you: I typically find it easier to read `map_err(TokenError::from)`